Readme.md

```diff
@@ -38,45 +38,45 @@ Why focus on soldering when you can play with the settings instead? With this ap
 - [X] Custom Preset buttons to allow quick change of user defined temperatures (PineSAM extra feature not available directly inside Pinecil).
 - [X] Backend runs locally on all major platforms/OS while the User Interface runs on your favorite browser.
 - [X] Temperature automatically adjusts based on °C or °F setting.
-- [X] Hide Minimum Voltage (per cell) when source is not battery.
+- [X] Power Settings: hide minimum voltage (per cell) when power source is not 3S-6S.
 
 ## Platforms
- | System  | Windows | MacOS | Linux | iOS | Android|
+ | System  | Windows | MacOS | :penguin: Linux | iOS | Android|
  | :-----: | :-----: | :---: | :---: | :-: | :----: |
  | backend |:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
  | UI      |:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
 <br>
 
-# Install Options
+# :desktop_computer: Install Options
 
 ## I. Using pre-made binaries
 
 1. Does not require installation of python or node.js
 2. Download latest [release](https://github.com/builder555/PineSAM/releases/latest) version for your OS system (Mac, Linux, Windows).
-2. **Mac/Linux**: extract and run `./start.sh` in terminal.
-3. **Windows**: 
-   * right click on the zip, properties> general, check box to Unblock.
-   * Run powershell as admin `.\start.ps1`
-   * Allow any windows displays pop-up warnings about "Serve.exe" and "main_sever.exe".".
-   * If you get [permission errors](https://lazyadmin.nl/powershell/running-scripts-is-disabled-on-this-system/)
-   
-```shell
-# set this one time in powershell
-Set-ExecutionPolicy RemoteSigned
-```
+3. **Mac/Linux**: extract and run `./start.sh` in terminal.
+4. **Windows**: 
+   * right click on the zip, properties> general, check box to Unblock, then extract.
+   * Run powershell as admin, cd to the pinesam folder and `.\start.ps1`
+   * Allow any windows pop-up warnings about "serve.exe" and "main_sever.exe".
+   * If you get errors in powershell, set the permissions to RemoteSigned to allow scripts (reference: [permission errors](https://lazyadmin.nl/powershell/running-scripts-is-disabled-on-this-system/)).
+   * ```Set-ExecutionPolicy RemoteSigned```
 
+5. Done! 🎉 now you can enjoy some soldering. Please ⭐ and share it with friends!
```

Comment:
this is probably unnecessary in the instructions, this readme is already getting pretty big.

Reply:
removed